### PR TITLE
[ISSUE-902][IMPROVEMENT][FOLLOWUP] LifecycleManager should reserve blacklist with  irrecoverable status

### LINKED ISSUE #902: [BUG] Blacklist should not clean worker that is shutting down to avoid retry on shutting down worker
### How to reproduce the bug?
Steps to reproduce the bug.

/cc @who-need-to-know

/assign @who-can-solve-this-bug

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### Related issues.
close #902

### How was this patch tested?

/cc @related-reviewer

/assign @main-reviewer

### CODE REVIEW

**Comment by AngersZhuuuu** [comment]
ping @waitinfuture

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala [hunk 1]
```diff
@@ -1332,6 +1333,14 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("3s")
 
+  val BLACKLIST_EXPIRED_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.blacklist.expired.timeout")
```
Comment (pan3793) [inline common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:1337]: `celeborn.worker.excluded.expired.timeout`

please avoid using white/black words

Comment (AngersZhuuuu) [reply common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:1337]: Done

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala [hunk 2]
```diff
@@ -1332,6 +1333,14 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("3s")
 
+  val EXCLUDED_WORKER_EXPIRED_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.excludedWorker.expired.timeout")
```
Comment (pan3793) [inline common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:1337]: we already have a `celeborn.worker.excluded.checkInterval` ...

Comment (AngersZhuuuu) [reply common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:1337]: > we already have a `celeborn.worker.excluded.checkInterval` ...

Different usage

Comment (pan3793) [reply common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:1337]: I mean the naming style

Comment (AngersZhuuuu) [reply common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:1337]: > I mean the naming style

Got it, updated

**Review by waitinfuture — APPROVED** [review]
LGTM, thanks!

**Pull request merged**